src/components/Breadcrumb.js

```diff
@@ -95,12 +95,11 @@ const Breadcrumb = props => {
             };
           }),
         });
-
         return (
           <nav {...rest}>
             <Ol>
-              {links.map(link => (
-                <CrumbLi key={link.label} {...link} />
+              {links.map((link, i, arr) => (
+                <CrumbLi key={link.label} {...link} path={(arr.length - 1 === i) ? null : link.path} /> 
```

Comment:
Here the spread out `{...link}` already contains `path` right? Does it now override the that path with the new path defined with `path={(arr.length ...`? Would it be more explicit that what is the path, if `link` was not deconstructed there, for example:
```js
<CrumbLi key={link.label} label={link.label} path={(arr.length - 1 === i) ? null : link.path} />
```